public/Jailbreak.Public/Mod/Rebel/IJihadC4Service.cs

```diff
@@ -0,0 +1,50 @@
+﻿
+using CounterStrikeSharp.API.Core;
+
+namespace Jailbreak.Public.Mod.Rebel;
+
+public interface IJihadC4Service
+{
+
+    /// <summary>
+    /// Calls the GiveJihadC4ToTerrorist function on a randomly chosen Terrorist 
+    /// at the start of each round, subject to certain conditions (such as if there
+    /// are more than 6 terrorists at the start of the round).
+    /// </summary>
+    public void GiveJihadC4ToRandomTerrorist();
+
+    /// <summary>
+    /// Give the "Jihad C4" to the specified Terrorist. This bomb is special and when 
+    /// the +use key is pressed when the bomb is equipped it will blow up and 
+    /// physically blast the Terrorists as well as damage the Guards in a certain radius.
```

Comment:
should physically push the terrorists.. i.e. they are affected by the bomb's blast!